src/ledger/LedgerManagerImpl.h

```diff
@@ -55,6 +55,8 @@ class LedgerManagerImpl : public LedgerManager
     std::unique_ptr<VirtualClock::time_point> mStartCatchup;
     medida::Timer& mCatchupDuration;
 
+    std::unique_ptr<LedgerCloseMeta> mLastLedgerMeta;
```

Comment:
I'd rename this to `mDelayedMetaFromLedger` (to match method name and clarify that it's delayed), and also add a comment relating this to the method and to the config flag.

Reply:
That's fine with me, although taking Jon's suggestion in https://github.com/stellar/stellar-core/pull/2982#discussion_r601832293 would change is meaning (it would spend some of its life as the meta from the ledger currently being closed, and might never be delayed as it would be used regardless of whether the config flag was set) and hence its choice of names.

Reply:
I did take Jon's suggestion, so that variable can now refer to current as well as delayed meta.  So I'm calling it `mNextMetaToEmit`.